README.md

```diff
@@ -32,7 +32,9 @@ async fn main() {
         }
     };
 
-    pin_mut!(s); // needed for iteration
+    // Stream must be pinned before iteration. It can be done with Box::pin,
```

Comment:
I would probably say something along the lines of stack pinning via `tokio::pin` is much more efficient but this can also be achieved by hoisting the item via `Box::pin`.